src/nvim/lib/klist.h

```diff
@@ -125,16 +124,11 @@
 
 #define kliter_t(name) kl1_##name
 #define klist_t(name) kl_##name##_t
-#define kl_val(iter) ((iter)->data)
-#define kl_next(iter) ((iter)->next)
-#define kl_begin(kl) ((kl)->head)
-#define kl_end(kl) ((kl)->tail)
```

Comment:
are these not useful as documentation? and, if we need any of these in the future?